job-dsls/jobs/kie/main/deploy_jobs.groovy

```diff
@@ -288,6 +281,11 @@ for (repoConfig in REPO_CONFIGS) {
                 }
             }
 
+            def downstreamRepos = get("downstreamRepos")
+            if (downstreamRepos) {
+                downstream(downstreamRepos, 'UNSTABLE')
```

Comment:
@mbiarnes I don't know where this `downstrea` method is defined. Can you please clarify?

Reply:
that is DSL plugin method @Ginxo see our jenkins APi <jenkins_URL>/plugin/job-dsl/api-viewer/index.html#method/javaposse.jobdsl.dsl.helpers.publisher.PublisherContext.downstream

Reply:
@Ginxo this downstream repos definition does:
![image](https://user-images.githubusercontent.com/3669059/130590280-e9fc5866-bd17-4793-9411-b1024938139f.png)
Please take a look at 
https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/mbiarnes/job/KIE/job/main/job/deployedRep/
All jobs are disabled! But this is this PR.